.typos.toml

```diff
@@ -1,5 +1,5 @@
 [files]
-extend-exclude = ["auto", "sys", "gobject-sys", "*.svg"]
+extend-exclude = ["auto", "sys", "gobject-sys", "*.svg", "glib/Gir.toml"]
```

Comment:
@bilelmoussaoui Is this fine with you?

Reply:
I would ask @Hofer-Julian

Reply:
I think the only alternative is ignoring "paranthasis" as a typo in general? Which I don't really like as an idea. I should add a comment explaining why we ignore that file though

Reply:
Yes, this looks fine to me